scrnaseq_hto.Rmd

```diff
@@ -16,17 +16,30 @@ params:
    
    # Input directory
    path_data: test_datasets/10x_pbmc_hto_GSE108313/counts
+  
+   # Metrics summary
+   stats:
    
    # Output directory
    path_out: test_datasets/10x_pbmc_hto_GSE108313/demultiplexed
    
    # HTO names
    #   HTOs have an ID that is included in the 'features.tsv' input file
-   #   We additionally ask for readable names that are used throughout this report 
+   #   We additionally ask for readable names that are used throughout this report (optional)
    #   This could look like this, where HTO1-3 are the IDs included raw dataset
-   #     param$hto.names = setNames(c("NameA", "NameB", "NameC"), c("HTO1", "HTO2", "HTO3"))
+   #    - without names: 'hto.names: c("HTO1", "HTO2", "HTO3")'
+   #    - with names: 'hto.names: !r setNames(c("NameA", "NameB", "NameC"), c("HTO1", "HTO2", "HTO3"))'
+   # 
    hto_names: !r setNames(c("htoA","htoB","htoC","htoD","htoE","htoF","htoG","htoH"), c("htoA","htoB","htoC","htoD","htoE","htoF","htoG","htoH"))
    
+   # HTO regular expression (alternative to HTO names)
+   #   Identify HTOs via a regular expression. Names cannot be provided here.
+   #   If specified will overrule hto_names
+   #hto_regex: "^hto"
```

Comment:
What will `"^hto"` do?

Reply:
All features starting with `hto` will be consered HTO.